diff --git a/node_modules/@tldraw/tldraw/src/lib/ui/components/StylePanel/StylePanel.tsx b/node_modules/@tldraw/tldraw/src/lib/ui/components/StylePanel/StylePanel.tsx
index 281ab80..556ecac 100644
--- a/node_modules/@tldraw/tldraw/src/lib/ui/components/StylePanel/StylePanel.tsx
+++ b/node_modules/@tldraw/tldraw/src/lib/ui/components/StylePanel/StylePanel.tsx
@@ -168,7 +168,7 @@ function CommonStylePickerSet({
 					/>
 				)}
 			</div>
-			{showPickers && (
+			{/* {showPickers && (
 				<div className="tlui-style-panel__section" aria-label="style panel styles">
 					{fill === undefined ? null : (
 						<ButtonPicker
@@ -201,7 +201,7 @@ function CommonStylePickerSet({
 						/>
 					)}
 				</div>
-			)}
+			)} */}
 		</>
 	)
 }
diff --git a/node_modules/@tldraw/tldraw/src/lib/ui/components/Toolbar/Toolbar.tsx b/node_modules/@tldraw/tldraw/src/lib/ui/components/Toolbar/Toolbar.tsx
index 65b9fcb..7fd116e 100644
--- a/node_modules/@tldraw/tldraw/src/lib/ui/components/Toolbar/Toolbar.tsx
+++ b/node_modules/@tldraw/tldraw/src/lib/ui/components/Toolbar/Toolbar.tsx
@@ -130,7 +130,7 @@ export const Toolbar = memo(function Toolbar() {
 					)}
 					<div
 						className={classNames('tlui-toolbar__tools', {
-							'tlui-toolbar__tools__mobile': breakpoint < 5,
+							'tlui-toolbar__tools__mobile': breakpoint > 5,
 						})}
 					>
 						{/* Select / Hand */}
@@ -208,7 +208,7 @@ export const Toolbar = memo(function Toolbar() {
 						)}
 					</div>
 				</div>
-				{breakpoint < 5 && !isReadonly && (
+				{breakpoint > 5 && !isReadonly && (
 					<div className="tlui-toolbar__tools">
 						<MobileStylePanel />
 					</div>
